CliClient/locales/es_ES.po

```diff
@@ -3287,4 +3287,4 @@ msgstr "Buscar"
 
 #, fuzzy
 #~ msgid "File system synchronisation target directory"
-#~ msgstr "Sincronización de sistema de archivos  en directorio objetivo"
+#~ msgstr "Sincronización de sistema de archivos en directorio objetivo"
```

Comment:
I think you did not use poedit. There are 2 things with this part:
- the translation is commented out (it was used previously, but is not in the current code)
- it is marked as fuzzy, which means the last translator or gettext was not sure about its accuracy and it is not used at all

So basically we can ignore this. But if you're sure it is accurate, you can remove the fuzzy line, and when this text should ever be used in the code again, we'd have a valid translation.

Reply:
Ok, let's ignore this for now. It's commented out anyway.